src/components/UserFollowingList.tsx

```diff
@@ -0,0 +1,68 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import NextLink from 'next/link';
+import { Button, Flex, Image, Text } from '@adobe/react-spectrum';
+
+import apiUrl from '../utils/apiUrl';
+import { ZetkinMembership } from '../types/zetkin';
+
+
+interface UserFollowingListProps {
+    following: ZetkinMembership[] | undefined;
+    onUnfollow: (orgId : number) => void;
+}
+
+const UserFollowingList = ({ following, onUnfollow } : UserFollowingListProps) : JSX.Element => {
+
+    if (!following || following.length === 0) {
+        return (
+            <Text data-testid="no-orgs-placeholder">
+                <Msg id="pages.myOrgs.orgsPlaceholder"/>
+            </Text>
+        );
+    }
+
+    return (
+        <Flex data-testid="following-list" direction="column">
+            { following.map(follow => (
+                <Flex
+                    key="follow.profile.id"
```

Comment:
I don't think this is doing what you were hoping to do when you wrote it. This means all organizations have the same key, the literal string _follow.profile.id_.